decidim-admin/app/views/decidim/admin/participatory_space_private_users/index.html.erb

```diff
@@ -15,10 +15,18 @@
         <table class="stack">
           <thead>
             <tr>
-              <th><%= t("models.assembly_user_role.fields.name", scope: "decidim.admin") %></th>
```

Comment:
Having an assembly_user_role in a participatory space?  This seemed like a bug, so I've fixed it.